op-batcher/batcher/channel_config_provider_test.go

```diff
@@ -71,6 +72,36 @@ func TestDynamicEthChannelConfig_ChannelConfig(t *testing.T) {
 			blobBaseFee:  1e9,
 			wantCalldata: true,
 		},
+		{
+			name:        "much-cheaper-blobs-l1-pectra",
+			tipCap:      1e3,
+			baseFee:     1e6,
+			blobBaseFee: 1,
+			isL1Pectra:  true,
+		},
+		{
+			name:        "close-cheaper-blobs-l1-pectra",
+			tipCap:      1e3,
+			baseFee:     1e6,
+			blobBaseFee: 390e5, // because of amortized fixed 21000 tx cost, blobs are still cheaper here...
+			isL1Pectra:  true,
+		},
+		{
+			name:         "close-cheaper-calldata-l1-pectra",
+			tipCap:       1e3,
+			baseFee:      1e6,
+			blobBaseFee:  410e5, // ...but then increasing the fee just a tiny bit makes blobs more expensive
```

Comment:
would be nice to try to get closer to the crossover point, like in the pre-Pectra tests. There it was `160e5` / `161e5`.

Reply:
[2575936](https://github.com/ethereum-optimism/optimism/pull/13870/commits/2575936bf21378a56b6d0647598bc37b7da88cd0)